mava/systems/tf/maddpg_v2/building.py

```diff
@@ -212,3 +225,12 @@ class MADDPGConfig:
 
 # Builder
 system_builder = Builder(config=config, components=system_components)
+
+# Create the launcher
+program = Launcher(multi_process=True, nodes_on_gpu=["trainer"], name="MADDPG")
```

Comment:
I liked the idea of having the program constructed outside of the builder but happy to chat about this some more. :)